src/components/LetniaHome.vue

```diff
@@ -31,3 +64,8 @@ a {
   color: #42b983;
 }
 </style>
+<style>
+button {
+  color: {{ color }};
+}
+</style>
```

Comment:
We should use newline at the end of a file. You can configure this in your VSC - Google "vscode add newline on save" :)